ansible/roles/test/files/ptftests/wr_arp.py

```diff
@@ -341,12 +368,10 @@ def runTest(self):
         m_pauses = { port:max(pauses[port]) for port in pauses.keys() if max(pauses[port]) > 25 }
         for port in m_pauses.keys():
             self.log("Port eth%d. Max pause in arp_response %d sec" % (port, int(m_pauses[port])))
-        print
```

Comment:
what is this print for?

Reply:
Maybe for a format, I keep the code in last diff.